ompi/mca/coll/han/coll_han.h

```diff
@@ -16,19 +17,23 @@
 
 #include "mpi.h"
 #include "ompi/mca/mca.h"
-#include "ompi/mca/coll/coll.h"
-#include "ompi/communicator/communicator.h"
-#include "ompi/include/mpi.h"
-#include "ompi/mca/coll/base/coll_base_functions.h"
-#include "opal/util/info.h"
-#include "ompi/op/op.h"
-#include "opal/runtime/opal_progress.h"
-#include "ompi/mca/pml/pml.h"
-#include "ompi/mca/coll/base/coll_tags.h"
+#include "opal/util/output.h"
 #include "ompi/mca/coll/base/coll_base_functions.h"
 #include "coll_han_trigger.h"
+#include "ompi/mca/coll/han/coll_han_dynamic.h" 
+
+BEGIN_C_DECLS
 
-BEGIN_C_DECLS typedef struct {
+/*
+ * Today;
+ * . only 2 modules available for intranode (low) level
+ * . only 2 modules available for internode (up) level
+ */
+
+#define COLL_FUTURE_LOW_MODULES 2
+#define COLL_FUTURE_UP_MODULES 2
+
```

Comment:
Maybe change FUTURE to HAN, FUTURE is the old name of this module.
#define COLL_HAN_LOW_MODULES 2
#define COLL_HAN_UP_MODULES 2